src/data/NetworkUpgradeSummaryData.ts

```diff
@@ -0,0 +1,231 @@
+interface NetworkUpgradeProps {
+  dateTimeAsString?: string
+  ethPriceInUSD?: number
+  waybackLink?: string
+  blockNumber?: number
+  epochNumber?: number
+  slotNumber?: number
+}
+
+export interface NetworkUpgradeSummaryDataProps {
+  [key: string]: NetworkUpgradeProps
+}
+
+const NetworkUpgradeSummaryData: NetworkUpgradeSummaryDataProps = {
+  shanghai: {
+    dateTimeAsString: "2023-04-12T22:27:35.000Z",
+    ethPriceInUSD: undefined,
+    waybackLink: undefined,
+    blockNumber: undefined,
+    epochNumber: undefined,
+    slotNumber: undefined,
```

Comment:
I would suggest we just remove the `undefined` declarations... The data schema is laid out above marking these as optional. @pettinarip Any thoughts/preferences?

Reply:
I would advocate keeping it just to make copy-pasting for new entries easier.

Reply:
Hrm, not critical of course, but not a big fan. Adds 53 lines just to leave a template for users. I think the interface should be the source people go to when seeing what properties to declare.